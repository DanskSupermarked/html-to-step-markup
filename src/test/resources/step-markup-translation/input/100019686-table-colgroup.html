<p><table width="696" cellspacing="0" cellpadding="0" border="0" style="WIDTH: 522pt; BORDER-COLLAPSE: collapse"> <colgroup> <col width="696" style="WIDTH: 522pt; mso-width-source: userset; mso-width-alt: 22272" /> </colgroup><tbody> <tr height="17" style="HEIGHT: 12.75pt"> <td width="696" height="17" style="BORDER-BOTTOM: #f0f0f0; BORDER-LEFT: #f0f0f0; BACKGROUND-COLOR: transparent; WIDTH: 522pt; HEIGHT: 12.75pt; BORDER-TOP: #f0f0f0; BORDER-RIGHT: #f0f0f0" class="xl65"><p><font face="Verdana">Rød Clip on lampe med motiv fra Cars 2. </font></p> <p><font face="Verdana">Har&nbsp;flexibel arm. </font></p> <p><font face="Verdana">P</font><font face="Verdana">erfekt som hygge- /læselampe på sengekanten.</font></p><p><font face="Verdana">Bruger 3 stk. AA batterier (Ekskl.)</font></p></td></tr></tbody></table></p>
