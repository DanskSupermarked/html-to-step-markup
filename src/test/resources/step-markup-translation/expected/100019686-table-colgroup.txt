Rød Clip on lampe med motiv fra Cars 2.

Har flexibel arm.

Perfekt som hygge- /læselampe på sengekanten.

Bruger 3 stk. AA batterier (Ekskl.)
